Sprint-1/exercises/paths.js

```diff
@@ -11,8 +11,16 @@
 
 const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
 const lastSlashIndex = filePath.lastIndexOf("/");
+    console.log(lastSlashIndex);
 const base = filePath.slice(lastSlashIndex + 1);
+    console.log(base);
 console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+
+const dirPart = filePath.substring(45, -1);
+    console.log(`The directory path of ${dirPart}`);
+
+const extPart = dirPart + "ext.txt";
```

Comment:
The "ext part" refers to the file extension of the file represented by the file path, which is the string after the dot in the base (including the dot).

So the file extension of "package.json" would be ".json".